packages/react-server/core/__tests__/integration/internalServerError/pages/InternalServerErrorException.js

```diff
@@ -3,8 +3,9 @@ import React from "react"
 export default class InternalServerErrorException {
 	handleRoute() {
 
-		// Not throwing an `Error` since this is easier to match.
-		throw "died";
+		// Need to throw a real error here, since CLS is going to
+		// assign a property to the error to track its source context.
+		throw new Error("died");
```

Comment:
If somebody in app code were to throw a string, it just gets no context, right (it doesn't blow up or anything?). I'm pretty sure we've done this before accidentally...